app/src/main/res/values/strings.xml

```diff
@@ -927,4 +927,11 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_pedestrian_crossing_island">"Does this pedestrian crossing have an island?"</string>
     <string name="quest_surface_detailed_answer_impossible_title">Describe surface</string>
     <string name="open_url">Open URL</string>
+    <string name="quest_tactile_paving_kerb_title">Is there tactile paving on this curb for this crossing?</string>
```

Comment:
"on this curb for this crossing" - why not simply "on this curb"?

Reply:
Because it is asking about tactile paving status just for crossing, not entire curb. But maybe it is communicated by quest element being a point.

Reply:
"on this curb here"?